lib/reviews.js

```diff
@@ -57,15 +57,23 @@ function parseFields($) {
 		var title = reviewContent.find('span[class=review-title]').text().trim();
 		var text = filterReviewText(reviewContent.text().trim(), title.length);
 
+		var developerComment = $(this).next().first();
```

Comment:
Why not .first()?
otherwise it returns with all replies and I think we wish only its relevant reply